core/lib/Thelia/Controller/Admin/HomeController.php

```diff
@@ -38,61 +50,99 @@ public function loadStatsAjaxAction()
             return $response;
         }
 
-        $data = new \stdClass();
-
-        $data->title = $this->getTranslator()->trans("Stats on %month/%year", array('%month' => $this->getRequest()->query->get('month', date('m')), '%year' => $this->getRequest()->query->get('year', date('Y'))));
-
-        /* sales */
-        $saleSeries = new \stdClass();
-        $saleSeries->color = $this->getRequest()->query->get('sales_color', '#adadad');
-        $saleSeries->data = OrderQuery::getMonthlySaleStats(
-            $this->getRequest()->query->get('month', date('m')),
-            $this->getRequest()->query->get('year', date('Y'))
-        );
-
-        /* new customers */
-        $newCustomerSeries = new \stdClass();
-        $newCustomerSeries->color = $this->getRequest()->query->get('customers_color', '#f39922');
-        $newCustomerSeries->data = CustomerQuery::getMonthlyNewCustomersStats(
-            $this->getRequest()->query->get('month', date('m')),
-            $this->getRequest()->query->get('year', date('Y'))
-        );
-
-        /* orders */
-        $orderSeries = new \stdClass();
-        $orderSeries->color = $this->getRequest()->query->get('orders_color', '#5cb85c');
-        $orderSeries->data = OrderQuery::getMonthlyOrdersStats(
-            $this->getRequest()->query->get('month', date('m')),
-            $this->getRequest()->query->get('year', date('Y'))
-        );
-
-        /* first order */
-        $firstOrderSeries = new \stdClass();
-        $firstOrderSeries->color = $this->getRequest()->query->get('first_orders_color', '#5bc0de');
-        $firstOrderSeries->data = OrderQuery::getFirstOrdersStats(
-            $this->getRequest()->query->get('month', date('m')),
-            $this->getRequest()->query->get('year', date('Y'))
-        );
-
-        /* cancelled orders */
-        $cancelledOrderSeries = new \stdClass();
-        $cancelledOrderSeries->color = $this->getRequest()->query->get('cancelled_orders_color', '#d9534f');
-        $cancelledOrderSeries->data = OrderQuery::getMonthlyOrdersStats(
-            $this->getRequest()->query->get('month', date('m')),
-            $this->getRequest()->query->get('year', date('Y')),
-            array(5)
-        );
-
-        $data->series = array(
-            $saleSeries,
-            $newCustomerSeries,
-            $orderSeries,
-            $firstOrderSeries,
-            $cancelledOrderSeries,
-        );
-
-        $json = json_encode($data);
-
-        return $this->jsonResponse($json);
+        $cacheExpire = intval(ConfigQuery::getAdminCacheHomeStatsTTL());
```

Comment:
with this static method you can control what is returned, please put your intval in the method ```getAdminCacheHomeStatsTTL()```